bitnami/kafka/values.yaml

```diff
@@ -146,541 +124,1017 @@ existingLog4jConfigMap: ""
 ## @param heapOpts Kafka Java Heap size
 ##
 heapOpts: -Xmx1024m -Xms1024m
-## @param deleteTopicEnable Switch to enable topic deletion or not
-##
-deleteTopicEnable: false
-## @param autoCreateTopicsEnable Switch to enable auto creation of topics. Enabling auto creation of topics not recommended for production or similar environments
-##
-autoCreateTopicsEnable: true
-## @param logFlushIntervalMessages The number of messages to accept before forcing a flush of data to disk
-##
-logFlushIntervalMessages: _10000
-## @param logFlushIntervalMs The maximum amount of time a message can sit in a log before we force a flush
-##
-logFlushIntervalMs: 1000
-## @param logRetentionBytes A size-based retention policy for logs
-##
-logRetentionBytes: _1073741824
-## @param logRetentionCheckIntervalMs The interval at which log segments are checked to see if they can be deleted
-##
-logRetentionCheckIntervalMs: 300000
-## @param logRetentionHours The minimum age of a log file to be eligible for deletion due to age
-##
-logRetentionHours: 168
-## @param logSegmentBytes The maximum size of a log segment file. When this size is reached a new log segment will be created
-##
-logSegmentBytes: _1073741824
-## @param logsDirs A comma separated list of directories in which kafka's log data is kept
-## ref: https://kafka.apache.org/documentation/#brokerconfigs_log.dirs
-##
-logsDirs: /bitnami/kafka/data
-## @param maxMessageBytes The largest record batch size allowed by Kafka
-##
-maxMessageBytes: _1000012
-## @param defaultReplicationFactor Default replication factors for automatically created topics
-##
-defaultReplicationFactor: 1
-## @param offsetsTopicReplicationFactor The replication factor for the offsets topic
-##
-offsetsTopicReplicationFactor: 1
-## @param transactionStateLogReplicationFactor The replication factor for the transaction topic
-##
-transactionStateLogReplicationFactor: 1
-## @param transactionStateLogMinIsr Overridden min.insync.replicas config for the transaction topic
-##
-transactionStateLogMinIsr: 1
-## @param numIoThreads The number of threads doing disk I/O
-##
-numIoThreads: 8
-## @param numNetworkThreads The number of threads handling network requests
+## @param interBrokerProtocolVersion Override the setting 'inter.broker.protocol.version' during the ZK migration.
```

Comment:
Putting it here but this is actually something for the statefulsets. Could you add the `runtimeClassName` value (check other charts)? This is a requirement of the NSA Kubernetes checklist

Reply:
Added 👍